tests/test_core.py

```diff
@@ -1032,8 +1032,8 @@ def test_persistant_state_integration(package, rf_method):
                 "Emissions|CO2|MAGICC Fossil and Industrial",
                 "Emissions|CO2|MAGICC AFOLU",
                 "Emissions|CH4|MAGICC Fossil and Industrial",
-                "Emissions|BC|MAGICC Fossil and Industrial",
-                "Emissions|SOx|MAGICC AFOLU",
+                "Emissions|Aerosols|Direct Effect|BC|MAGICC Fossil and Industrial",
+                "Emissions|Aerosols|Direct Effect|SOx|MAGICC AFOLU",
```

Comment:
```suggestion
                "Emissions|SOx|MAGICC AFOLU",
```

Direct effect only applies to radiative forcing, everything else shouldn't change